akka-http-core/src/main/scala/akka/http/javadsl/ClientTransport.scala

```diff
@@ -33,6 +33,7 @@ abstract class ClientTransport {
  */
 @ApiMayChange
 object ClientTransport {
+  implicit val ct = akka.http.impl.util.JavaMappingCore.ClientTransport
```

Comment:
For some reason, if I import `JavaMappingCore._` the implicit objects are not picked, I need to manually put them into scope

Reply:
It's probably because of name clashes. Maybe renaming the instances would be enough for imports to work.